cbt.go

```diff
@@ -1363,6 +1379,16 @@ func doRead(ctx context.Context, args ...string) {
 		}
 		filters = append(filters, columnFilters)
 	}
+	keysOnly := false
```

Comment:
Is the duplication here what you were references in the commit message?

Reply:
Yep! I think there's an opportunity to refactor the function signatures here so as to reduce the duplication of code.

Reply:
nit: `var keysOnly bool`.

Reply:
Done